cordova/config.xml

```diff
@@ -81,6 +82,7 @@
     <engine name="ios" spec="https://github.com/apache/cordova-ios.git#4.5.5" />
     <engine name="android" spec="~6.3.0" />
     <engine name="windows" spec="~4.3.1" />
+    <plugin name="com.testfairy.cordova-plugin" spec="https://github.com/testfairy/testfairy-cordova-plugin.git" />
```

Comment:
I like the idea here of adding testfairy-cordova-plugin to the config.xml but we only want this plugin installed for staging environments. What happens in production?

Reply:
It'll install regardless... I was facing some issues with installation in rakefile so moved it to xml.
Will try to change it back to RakeFile